models/users.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const userSchema = new mongoose.Schema({
+    name: {
+        type  :String,
+        required : true
+    },
+    email: {
+        type  :String,
+        required : true
+    },
+    password: {
+        type : String,
+        required : true;
+    },
+    date: {
```

Comment:
change date to something like "createdon" , something meaningful.

Reply:
What does "createdon" here mean ?